requirements.txt

```diff
@@ -2,4 +2,5 @@ requests~=2.32.3
 urllib3~=2.2.2
 scikit-learn~=1.5.0
 Flask~=3.0.3
-Jinja2~=3.1.4
\ No newline at end of file
+Jinja2~=3.1.4
+ipdb~=0.13.13
```

Comment:
I pulled the code down and checked by running without installing this, and this requirement for ipdb is not needed for this PR, that I can tell. Please correct me if I'm wrong.

To save me from having to update the 3rd party license readme and pull in this license, could you remove that?

Reply:
Yeah, I'll remove it. If you haven't tried ipdb for debugging, I recommend it, but no need to force it on everyone.